apps/nextjs-website/react-components/components/Footer/assets/FundedByNextGenerationEU/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { theme } from '@pagopa/mui-italia';
+import { StyledSvg } from '../Assets.helpers';
+import {
+  SvgFilled,
+  SvgOutline,
+  SvgColorDark,
+  SvgColorLight,
+} from './partials';
+
+type NextGeneratioNEUVariants = 'filled' | 'outline' | 'color';
+type NextGenerationEUColors = 'light' | 'dark' | 'pantone';
+
+interface FundedByNextGenerationEUProps {
+  /** Provides a human-readable title for the element that contains it. */
+  title?: string;
+  /** Width of the component. Height is set automatically. */
+  size?: number;
+  /** The variant to use. If the color variant is set,
+   * the color property is ignored. */
+  variant?: NextGeneratioNEUVariants;
```

Comment:
```suggestion
  variant?: NextGenerationEUVariants;
```
Consequence of above change.